include/lo2s/trace/trace.hpp

```diff
@@ -169,6 +170,32 @@ class Trace
         return cpuid_metric_class_;
     }
 
+    otf2::definition::metric_class topdown_metric_class()
+    {
+        if (!topdown_metric_class_)
+        {
+            topdown_metric_class_ = registry_.create<otf2::definition::metric_class>(
+                otf2::common::metric_occurence::async, otf2::common::recorder_kind::abstract);
+            topdown_metric_class_->add_member(metric_member(
+                "slots", "Number of UOPS slots", otf2::common::metric_mode::absolute_point,
+                otf2::common::type::int64, "slots"));
```

Comment:
the second `"slots"` is the unit of the metric. uops seems to be the obvious choice.